docs/learn/resource-materials/glossary.md

```diff
@@ -1,34 +1,196 @@
+- **Branch (IOTA 2.0):** A version of the ledger that temporarily coexists with other versions, each spawned by conflicting transactions.
+- **Bee:** Node software developed by the IOTA foundation using the Rust programming language.
+- **Bootstrapping attack:** An attack in which a node downloads malicious snapshot files, including invalid transactions and balances.
 
+## C
+
+- **Curl:** This is the main hash function currently in use. It is based on the “sponge” construction of the Keccak inventors (SHA-3). 
+- **Confirmed:** Confirmed transactions. In IOTA 1.5, transactions are still confirmed by the coordinator (milestones). 
+- **CTPS:** Confirmed transaction per second.
+- **Cumulative Weight:** A system for valuing transactions. Each additional transaction that references a transaction increases its cumulative weight. When tips are selected, a path through transactions that has a higher cumulative weight is preferred.
+- **CommNet:** The CommNet is a test-only network and is similar to the Testnet except that it is maintained by the IOTA community. The Hornet nodes in the CommNet continue to use a coordinator operated by the IOTA community.
+- **Chronicle:** Official permanode solution of the IOTA Foundation. It allows to store all transactions reaching a node in a distributed database that is secure and scales well. Chronicle is used to store the Tangle’s unlimited data flow and make it queryable. In other words, permanence allows the entire history of the Tangle to be stored indefinitely and makes that data easily accessible.
+- **Consensus:** Agreement on a specific date or value in distributed multi-agent systems, in the presence of faulty processes. 
+- **Coordinator (only up to IOTA 2.0):** A trusted entity, as protection against malicious transactions. The Tangle is not yet a final product, it is still in beta. The network currently relies on a kind of shield, the so-called coordinator. It is open-source and runs on a Hornet node. The COO acts as a centralized, voluntary, and temporary alternative consensus mechanism for the Tangle. To do this, the COO sends honest transactions to the full nodes at regular intervals. These packets contain a signed message with no value, called a milestone. The full nodes in the Tangle consider a transaction as confirmed only if it is approved by a milestone. Important: The coordinator can only confirm transactions, but he cannot bypass the consensus rules. To create, freeze or steal tokens is not possible for him. This fixed rule and the COO address is hardcoded on each full node, so the coordinator’s influence on the tangle is very limited, since the tangle is also constantly monitored by all the other full nodes. > The Coo will be switched off with the IOTA 2.0 upgrade.
+- **Communication Layer (IOTA 2.0):** This layer stores and communicates information. This layer contains the distributed ledger or tangle. The rate control and timestamps are also located in this layer.
```

Comment:
Agree, also remove the communication layer point.. it is not used as a term in IOTA 2.0 anymore (where we seem to end up with "branch" instead of layer)